library/cwm/src/lib/cwm/widget.rb

```diff
@@ -258,6 +258,60 @@ def validate_wrapper(_widget_id, _event)
     end
   end
 
+  # Wrapper when combination of old hash based CWM definition needed to be used
+  # with new object based one. Useful e.g. when content is provided by other
+  # module like CWMFirewallInterfaces.CreateOpenFirewallWidget
+  # @note it does not support common methods and helpers from abstract widget
```

Comment:
To be honest, I would like to keep object hierarchy clean. Another ancestor can be a bit confusing. What name you have in mind? It can maybe help if it have good name.